realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/AuthTests.java

```diff
@@ -824,4 +824,120 @@ public void onError(ObjectServerError error) {
             }
         });
     }
+
+    @Test
+    @RunTestInLooperThread
+    @Ignore("Depends on https://github.com/realm/realm-java/pull/5909")
+    public void requestPasswordResetAsync() {
+        String email = "foo@bar.baz";
+        UserFactory.createUser(email).logOut();
+
+        // Currently no easy way to see if we actually get an email.
```

Comment:
In theory, we can if we configure ROS with the emailHandler to point to `https://ethereal.email` then query the SMTP server to fetch the different emails to retrieve the different tokens, but I guess this is out of scope ...